### PR TITLE
no_grad scope for debug

### PR DESCRIPTION
**Pull request by FelixAbrahamsson**
We don't need to save gradient information when debugging, it consumes more memory.

### CODE REVIEW

**Review by samedii — APPROVED** [review]
Sure this makes sense. I would suggest using torch.no_grad as a decorator instead though :)
